From 4f430217daf3e394354e9d6273c6151788fb959c Mon Sep 17 00:00:00 2001
From: Rob Herring <rob.herring@calxeda.com>
Date: Fri, 22 Feb 2013 12:39:30 -0600
Subject: [PATCH] cpuidle: calxeda: disable power gating by default
Patch-Mainline: No, will be soon

This only works with the right firmware, so disable by default.

Signed-off-by: Rob Herring <rob.herring@calxeda.com>
Signed-off-by: Alexander Graf <agraf@suse.de>
---
 drivers/cpuidle/cpuidle-calxeda.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/cpuidle/cpuidle-calxeda.c b/drivers/cpuidle/cpuidle-calxeda.c
index e1aab38..b59cef9 100644
--- a/drivers/cpuidle/cpuidle-calxeda.c
+++ b/drivers/cpuidle/cpuidle-calxeda.c
@@ -141,6 +141,7 @@ static int __init calxeda_cpuidle_init(void)
 		dev = per_cpu_ptr(calxeda_idle_cpuidle_devices, cpu_id);
 		dev->cpu = cpu_id;
 		dev->state_count = drv->state_count;
+		dev->states_usage[1].disable = 1;
 
 		ret = cpuidle_register_device(dev);
 		if (ret) {
-- 
1.7.4.1

